src/components/course-calendar/duration.cjsx

```diff
@@ -183,6 +196,14 @@ CourseDuration = React.createClass
     if calcedHeight > rangeData.dayHeight
       rangeData.dayHeight = calcedHeight
 
+  _getSimplePlan: (plan) ->
+    simplePlan = _.omit(plan, 'due_at', 'opens_at', 'duration', 'durationAsWeeks')
```

Comment:
Would specifically including, rather than omitting be shorter? It is not clear why these fields specifically are excluded and which ones are included.